cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -128,6 +151,67 @@ void __wrap_system_restart_local() {
 
     ets_install_putc1(&uart_write_char_d);
 
+    /*
+      Check for Exceptions mapped into User Exception.
+    */
+    if (rst_info.reason == REASON_EXCEPTION_RST && rst_info.exccause == 0) {
+#if defined(DEBUG_ESP_PORT) || defined(DEBUG_ESP_EXCEPTIONS)
+        if ((rst_info.epc1 - 11u) == (uint32_t)_DoubleExceptionVector) {
```

Comment:
Why 11? what does that magic number mean?

Reply:
That is the offset into the `_DoubleExceptionVector` where the `ill` instruction was executed. I found a better way to deal with that. I'll push later,